README.md

```diff
@@ -1,12 +1,18 @@
 # Место (backend at local "express" server)
 Yandex Praktikum single page testwork for Sprint 13 by Max Konovalov
 
+v0.3.0 - First "production" version. Logging and request validation via Joi added.
+
+v0.2.2 - Review bug fixed, helmet module added
+
 v0.2.0 - Authorization via token included
+
 v0.2.1 - Some bug fixed
-v0.2.2 - Review bug fixed, helmet module added
 
 ## About
-This testwork is about launch and configure local server via "Express" by Node JS for frontend part of Mesto project ([Sprint 11](https://konovaly4.github.io/Praktikum_sprint_11.github.io/)). This version includes MongoDB and Mongoose to contain collections of users and cards.
+This testwork is about launch and configure server via "Express" by Node JS for frontend part of Mesto project ([Sprint 11](https://konovaly4.github.io/Praktikum_sprint_11.github.io/)). This version includes MongoDB and Mongoose to contain collections of users and cards. Server provided by Yandex Cloud service.
+
+Project is avaliable at domain http://konovalovalov-backend.tk, public IP: 84.201.148.214
```

Comment:
Нужно исправить: невозможно подключиться к публичному серверу по домену и ip.
Все запросы делаю в Postman. Поэтому ошибки будут согласно его представлению.
Запрос `POST http://konovalovalov-backend.tk/signup`.
Ответ:
```
Could not get any response
There was an error connecting to http://konovalovalov-backend.tk/signup.
Why this might have happened:
The server couldn't send a response:
Ensure that the backend is working properly
Self-signed SSL certificates are being blocked:
Fix this by turning off 'SSL certificate verification' in Settings > General
Proxy configured incorrectly
Ensure that proxy is configured correctly in Settings > Proxy
Request timeout:
Change request timeout in Settings > General
```
Запрос по протоколу `HTTPS` отдает тот же ответ.

Запрос `POST http://84.201.148.214/signup`.
Ответ с кодом 404:
```html
<html>

<head>
	<title>404 Not Found</title>
</head>

<body bgcolor="white">
	<center>
		<h1>404 Not Found</h1>
	</center>
	<hr>
	<center>nginx/1.14.0 (Ubuntu)</center>
</body>

</html>
```

Ожидается, что сервер будет работать также как и при запуске локально.

Reply:
Вроде все поправил (пересоздал ВМ), должна быть возможность подключиться по адресу и IP в README.md или тут:
http://konovalov-backend.tk
IP: 84.201.170.64
(Прошлую проблему с подключением так и не разобрал, к моему сожалению, хотя все работало когда я тестировал запросы...)